README.md

```diff
@@ -1,2 +1,21 @@
 # highscore-database
 An API allowing for the saving and retrieval of game high scores
+
+# Dependencies
+
+We are building our api with [FastAPI](https://fastapi.tiangolo.com/)
+
+# Setup
+
+
+### Windows
+```
+environment\Scripts\activate.bat
+pip install -r requirements.txt
+```
+
+### Linux
+```
+source environment\Scripts\activate
```

Comment:
Backslashes (\\) are Windows only. The rest of the world use forward slashes (/)